src/lib/components/CommunitySelectionModal/CommunitySelectionSearch.js

```diff
@@ -37,13 +35,27 @@ export class CommunitySelectionSearch extends Component {
 
     this.state = {
       selectedEndpoint: defaultEndpoint,
+      overriddenComponents: props.overriddenComponents,
     };
   }
 
+  componentDidMount() {
+    const { appName } = this.props;
+    const { overriddenComponents } = this.state;
+    if (_isEmpty(overriddenComponents)) {
+      const defaultOverriddenComponents = {
+        [`${appName}.ResultsList.item`]: CommunityListItem,
+      };
+      this.setState({
+        overriddenComponents: defaultOverriddenComponents,
```

Comment:
why we need overriddenComponents inside the state?

Reply:
Because the cmp overrides cmps for RSK itself, see here: https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/components/CommunitySelectionModal/CommunitySelectionSearch.js#L60

Initially we want to tune this and override this cmps to display the items differently, as in the mockups there were differences between the modal to select communities in the landing page and the modal from the deposit form, however we decided the keep them the same in the demo, so we could actually remove this.